feat(curriculum): Add English tasks B1 Block 6 (#57565)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/676147de99ef019cee551581.md

---
id: 676147de99ef019cee551581
title: Task 109
challengeType: 19
dashedName: task-109
---

<!-- READING -->

# --instructions--

Read the text and answer the question below.

# --questions--

## --text--

`Dear Team,`

`As the summit is near and many of you will be traveling for work, it’s important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`

`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we’ve shared. They include simple steps to keep your data safe while traveling.`

`We’ve scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`

`Stay safe,`

> `Your Security Team`

Reviewer: Added introduction and changed apostrophes.

Why is it important to stay secure online while traveling for work?

## --answers--

To avoid delays during the trip.

### --feedback--

> The email does not mention delays as a reason for staying secure online. Focus on the risk of data being stolen.

Reviewer: ```suggestion
The email does not mention delays as a reason for staying secure online.
```
This feedback gives away the answer.

---

To use public Wi-Fi safely.

### --feedback--

The email warns against using public Wi-Fi for sensitive tasks, not about using it safely.

---

To protect sensitive data from being stolen.

---

To connect to the summit network faster.

### --feedback--

> The email does not mention anything about connecting to a summit network. The focus is on protecting sensitive information.

Reviewer: ```suggestion
The email does not mention anything about connecting to a summit network.
```
This feedback gives away the answer.

## --video-solution--

3

# --explanation--

> To find the correct answer, focus on the email’s warning about Man-in-the-Middle attacks.

Reviewer: ```suggestion
To find the correct answer, focus on the email's warning about Man-in-the-Middle attacks.
```
